### PR TITLE
Change for LANSA Japan AWS account

### PR DESCRIPTION
**Pull request by Nom3k**
This reuqest from Kei Nomiyama LANSA Japan.

### CODE REVIEW

File: scripts/Init-Baking-Vars.ps1 [hunk 1]
```diff
@@ -26,7 +26,7 @@ if ( test-path variable:KeyPairName) {
 }
 
 if ( test-path variable:KeyPairPath) {
-    $script:keypairfile = $KeyPairPath
+    $script:keypairfile = "$KeyPairPath\$KeyPairName.pem"
```
Comment (robe070) [inline scripts/Init-Baking-Vars.ps1:29]: Why don't you just enter the keypair file name with ".pem" on the end?

File: scripts/bake-jpn-ami.ps1 [hunk 2]
```diff
@@ -54,7 +54,7 @@ param (
 
     [Parameter(Mandatory=$false)]
     [string]
-    $Language='ENG',
+    $Language='JPN',
```
Comment (robe070) [inline scripts/bake-jpn-ami.ps1:57]: PLease pass the language parameter in to this script as JPN. So make the change to bake-scalable-aws-jpn.ps1

Comment (robe070) [reply scripts/bake-jpn-ami.ps1:57]: Kei, please note that I have reverted my changes to `bake-scalable-aws-jpn.ps1` file and created a new file for me to use `bake-scalable-aws-jpn-full.ps1`.

So make sure you PULL MY CHANGES before you submit your PR again.

File: scripts/Init-Baking-Vars.ps1 (hunk 1 above)
Comment (Nom3k) [reply scripts/Init-Baking-Vars.ps1:29]: The power shell script was failing in authentication.
I was able to clear the authentication by using the keypair path and keypair file name with extension.

Comment (robe070) [reply scripts/Init-Baking-Vars.ps1:29]: I'll need to look more deeply into this on Tuesday

Comment (robe070) [reply scripts/Init-Baking-Vars.ps1:29]: And, the ".pem' should be passed in, not presumed. $KeyPairName = 'id_isa.pem', or whatever in the calling script.

**Pull request merged**